setup.cfg

```diff
@@ -19,13 +19,13 @@ markers =
 [pep8]
 max-line-length = 99
 ignore = E116,E121,E123,E126,E133,E226,E241,E242,E302,E704,E731
-exclude = build/*,.tox/*,tests/*,ve/*,*/_vendor/*,conda/compat.py,conda_env/*
+exclude = build/*,.tox/*,tests/*,ve/*,*/_vendor/*,conda/compat.py,conda_env/compat.py
```

Comment:
You always ask question you already know the answer to Alan :-p

Yes, that's where I usually put a project's dev/working virtualenv. Need to put it somewhere. Guess we could document it better though.